package.json

```diff
@@ -98,7 +98,7 @@
     "@hot-loader/react-dom": "^16.13",
```

Comment:
Collection context menu not right-aligned

![image](https://user-images.githubusercontent.com/64950861/148082965-2c23158d-9e95-420e-a609-9aaff8790808.png)

Reply:
Color whack-a-mole

![image](https://user-images.githubusercontent.com/64950861/148083114-f577b984-906d-4616-82a4-29f061ea962c.png)

Reply:
I had no custom playlist :man_facepalming: Fixed and tested on desktop & mobile.